commands/version.go

```diff
@@ -41,9 +49,62 @@ func runVersion(cmd *cobra.Command, args []string) {
 		fmt.Println(version.BuildVersion())
 	} else {
 		printFiglet()
-		fmt.Printf("Commit: %s\n", version.GitCommit)
-		fmt.Printf("Version: %s\n", version.BuildVersion())
+		fmt.Printf("CLI commit: %s\n", version.GitCommit)
+		fmt.Printf("CLI version: %s\n", version.BuildVersion())
+		printServerVersions()
 	}
+
+}
+
+func printServerVersions() {
+
+	var services stack.Services
+	var gatewayAddress string
+	var yamlGateway string
+	if len(yamlFile) > 0 {
+		parsedServices, err := stack.ParseYAMLFile(yamlFile, regex, filter)
+		if err == nil && parsedServices != nil {
+			services = *parsedServices
+			yamlGateway = services.Provider.GatewayURL
+		}
+	}
+
+	gatewayAddress = getGatewayURL(gateway, defaultGateway, yamlGateway, os.Getenv(openFaaSURLEnvironment))
+
+	timeout := 2 * time.Second
+	client := proxy.MakeHTTPClient(&timeout)
+	response, err := client.Get(gatewayAddress + "/system/info")
+	if err != nil {
+		fmt.Printf("Warning could not contact gateway for version information on %s %s\n", gatewayAddress+"/system/info", err.Error())
+		return
+	}
+
+	info := make(map[string]interface{})
+	upstreamBody, _ := ioutil.ReadAll(response.Body)
+	err = json.Unmarshal(upstreamBody, &info)
+	if err != nil {
+		fmt.Printf("Error during unmarshal of body %s %s\n", upstreamBody, err.Error())
+		return
+	}
+
+	version, sha, commit := getGatewayDetails(info)
+	printGatewayDetails(gatewayAddress, version, sha, commit)
+
+	name, orchestration, sha, version := getProviderDetails(info)
+	fmt.Printf("- Provider \n\tname: %s \n\torchestration: %s \n\tversion: %s \n\tsha: %s\n",
```

Comment:
You can split this up with the ` character. It will mean you can write:

Printf(`line1
 line 2
 line 3
line 4
 line 5
`)
etc

Reply:
oh yeah good point :+1:

Reply:
That didn't appear as intended. Basically instead of " you can use ` and then wrap the lines